lib/util/cache/repository/index.spec.ts

```diff
@@ -38,28 +38,16 @@ describe('lib/util/cache/repository', () => {
     });
   });
   it('reads from cache and finalizes', async () => {
-    fs.readFile.mockResolvedValueOnce('{"repository":"abc/def"}' as any);
-    await repositoryCache.initialize({
-      ...config,
-      repositoryCache: 'enabled',
-    });
-    await repositoryCache.finalize();
-    expect(fs.readFile.mock.calls).toHaveLength(1);
-    expect(fs.outputFile.mock.calls).toHaveLength(1);
-  });
-  it('migrates', async () => {
```

Comment:
Caches will be invalidated whenever the revision changes, so any existing migrations aren't necessary.